enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/scenarios/ClusterOverviewIT.java

```diff
@@ -171,20 +158,22 @@ public void shouldDiscoverReadReplicasAfterRestartingCores() throws Exception
     public void shouldDiscoverNewCoreMembers() throws Exception
     {
         // given
-        clusterRule.withNumberOfCoreMembers( 3 );
+        int initialCoreMembers = 3;
+        clusterRule.withNumberOfCoreMembers( initialCoreMembers );
         clusterRule.withNumberOfReadReplicas( 0 );
 
         Cluster cluster = clusterRule.startCluster();
 
         // when
-        cluster.addCoreMemberWithId( 3 ).start();
-        cluster.addCoreMemberWithId( 4 ).start();
+        int extraCoreMembers = 2;
+        int finalCoreMembers = initialCoreMembers + 2;
+        IntStream.range( 0, extraCoreMembers ).forEach( idx -> cluster.addCoreMemberWithId( initialCoreMembers + idx ).start() );
```

Comment:
If we're going to go this way: "documenting" with variable names, should we factor out / replicate to make it consistent across the other tests in this IT?

Reply:
Baby steps... I'll update the other tests just as soon as I care about them :)

Reply:
although the `2` on line 169 looks like it should be `extraCoreMembers`...